src/library/scala/math/Numeric.scala

```diff
@@ -134,7 +134,7 @@ object Numeric {
     def div(x: Float, y: Float): Float = x / y
   }
   trait FloatAsIfIntegral extends FloatIsConflicted with Integral[Float] {
-    def quot(x: Float, y: Float): Float = (BigDecimal(x) / BigDecimal(y)).floatValue
```

Comment:
Since we're changing the current behavior of `FloatAsIfIntegral`, this needs its own unit test. I suggest that you use `bigDecimalTest.scala` to add the tests for `quot`.